src/benchmark.cc

```diff
@@ -174,7 +186,8 @@ void RunInThread(const benchmark::internal::Benchmark::Instance* b,
     results.manual_time_used += timer.manual_time_used();
     results.bytes_processed += st.bytes_processed();
     results.items_processed += st.items_processed();
-    results.complexity_n += st.complexity_length_n();
+    // explicitly avoid warning on Visual C++ about truncating int64_t
+    results.complexity_n += static_cast<int>(st.complexity_length_n());
```

Comment:
this was fixed in another commit so you shouldn't need it any more (`complexity_n` is 64-bit).

Reply:
Gotcha: will remove!